libraries/botbuilder-dialogs-adaptive/src/input/choiceInput.ts

```diff
@@ -40,6 +40,14 @@ export class ChoiceInput extends DialogCommand implements DialogDependencies {
     /**
      * Activity to send the user.
      */
+    public set prompt(value: string|Partial<Activity>) {
```

Comment:
This was needed because the original implementation was constructor based, but we need property based setting of this from a string.